README.md

```diff
@@ -1,6 +1,6 @@
 # Account Abstraction demo app
 
-[The Safe{Core} Account Abstraction SDK](https://github.com/safe-global/account-abstraction-sdk) allows builders to add account abstraction functionality into their apps. This demo is an example on how to use our different packages (Auth Kit, OnRamp Kit & Relay Kit).
+[The Safe{Core} SDK](https://github.com/safe-global/safe-core-sdk) provides a suite of tools to interact with the Safe ecosystem. This demo app is an example of how to use our different packages (Auth Kit, OnRamp Kit & Relay Kit).
```

Comment:
Not sure about `Safe ecosystem`

Reply:
I think that makes sense to keep the old description!